src/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  padding-left: 16px;
+  padding-right: 16px;
+  align-items: center;
+  border-radius: 5px;
+  justify-content: center;
+  width: 200px;
+}
+
+.card__imac_image {
+  margin-top: 32px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product_name {
+  width: 100%;
+  margin: 40px auto 4px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__product_code {
+  width: 100%;
+  color: #616070;
+  font-size: 10px;
+}
+
+.card__block_rating {
+  margin-top: 16px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__reviews {
+  display: flex;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__block_price {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price_meaning {
+  display: flex;
+  font-size: 16px;
+  font-weight: 700;
+  width: min-content;
```

Comment:
redundant

Reply:
fix the same issue everywhere, not only exactly where the comment was written (for the feature)
 https://github.com/mate-academy/layout_product-cards/pull/2081#discussion_r1010668692